fossunited/api/dashboard.py

```diff
@@ -8,7 +8,7 @@
 
 @frappe.whitelist(allow_guest=True)
 def get_event(name: str) -> dict:
-    return frappe.get_doc("FOSS Chapter Event", name)
+    return frappe.get_doc("FOSS Chapter Event", name).as_dict()
```

Comment:
Why?

Reply:
Is it better to return dict rather than object or it doesn't make a difference?

Reply:
Does not make a difference, Frappe will do that for you.